OAuth 2.0 token exchange docs (#5181)
ydb-platform/ydb: ydb/docs/en/core/reference/ydb-cli/_includes/auth/env_oauth2.md

> - Otherwise, if the value of the `YDB_OAUTH2_KEY_FILE` environment variable is set, the **OAuth 2.0 token exchange** authentication mode is used and the parameters for token exchange is taken from the [json-file](../../reference/ydb-sdk/auth#oauth2-key-file-format) whose name is specified in this variable.
>     - For setting token endpoint option `--iam-endpoint` option is used (via {{ ydb-short-name }} CLI option, profile or environment variable) in `SCHEMA://HOST:PORT/PATH` format
>     - [Detailed information about OAuth 2.0 token exchange protocol](https://www.rfc-editor.org/rfc/rfc8693)

Reviewer: ```suggestion
- Otherwise, if the `YDB_OAUTH2_KEY_FILE` environment variable value is set, the [**OAuth 2.0 token exchange**](https://www.rfc-editor.org/rfc/rfc8693) authentication mode is used, and the token exchange parameters are taken from a [JSON file](../../reference/ydb-sdk/auth#oauth2-key-file-format) specified in this variable. The `--iam-endpoint` option is used to set the token exchange endpoint in the `<schema>://<host>:<port>/<path>` format (via a {{ ydb-short-name }} CLI option, profile, or environment variable).
```
